get_bullseye.sh

```diff
@@ -7,10 +7,10 @@ dir_out="bullsye_dir.html"
 
 # Need to get a directory listing of files for download
 curl --location --retry 10 --retry-max-time 60 --silent --show-error \
-  "${bullseye_url}" -o $dir_out
+  "${bullseye_url}.html" -o $dir_out
 
 # Extract the filename from the listing
-dnl_fname="$(sed -ne '/Linux-x64/s/.*href="\(.*\)\".*<\/a>.*/\1/p' $dir_out)"
+dnl_fname="$(sed -ne '/Linux-x64/s/.*href=download\/\([^>]*\).*/\1/p' $dir_out)"
```

Comment:
FWIW, you _could_ eliminate the file in the middle with (I think the suggestion is being confused by including unchanged lines -- a GH bug/deficiency IMO):

```suggestion
dnl_fname="$(sed -ne '/Linux-x64/s/.*href=download\/\([^>]*\).*/\1/p' <(curl --location --retry 10 --retry-max-time 60 --silent --show-error "${bullseye_url}.html"))"
```